pkgs/tools/misc/ytmdl/default.nix

```
@@ -15,6 +16,15 @@ python3.pkgs.buildPythonApplication rec {
     sha256 = "1jpd5zhqg2m9vjjjw4mgzb594q1v1pq1yl65py6kw42bq9w5yl5p";
   };
 
+  patches = [
+    # Fixes ytmdl#188 and is only needed until the next major release after 2021.06.26
+    (fetchpatch {
+      url = "https://github.com/deepjyoti30/ytmdl/compare/2021.06.26...37ba821d9692249c1fa563505cf60bd11b8e209e.patch";
```

Please use this syntax, so it is obvious you are applying a single commit
```suggestion
      url = "https://github.com/deepjyoti30/ytmdl/commit/37ba821d9692249c1fa563505cf60bd11b8e209e.patch";
```